wgsl/wgsl_spec_style_guide.md

```diff
@@ -0,0 +1,79 @@
+# WSGL spec writing style guide
+
+Goal:  Avoid possibly being misunderstood.
+*   Tradeoff: The text might read as stilted.  **Precision is better than flair.**
+
+Prefer short sentences.
+
+Be clear about which agent is doing what:
+*   The _shader author_ is the person writing the WGSL code for use in their application
+*   The _implementation_ is the thing that runs the application, processing the WGSL code.
+    *   Implicitly, the implementation is a hardware-software combination, and we don’t care about the boundary between them.
+    *   Avoid overly specific terms like “hardware” or “instructions”.
```

Comment:
In general, it seems like this guideline is focused on the *normative* text, which is a subset of the whole spec.
Let's mention it here explicitly, perhaps?